index.bs

```diff
@@ -193,13 +194,69 @@ The <dfn for="DOM/Document">parseHTML</dfn>(|html|, |options|) method steps are:
    Note: Since |document| does not have a browsing context, scripting is disabled.
 1. Set |document|'s [=allow declarative shadow roots=] to true.
 1. [=Parse HTML=] from a string given |document| and |html|.
-1. Let |config| be the result of calling [=canonicalize a configuration=] on
-   |options|["`sanitizer`"] and true.
+1. Let |config| be the result of calling [=get a sanitizer config from options=]
+   with |options| and true.
 1. Call [=sanitize=] on |document|'s [=tree/root|root node=] with |config|.
 1. Return |document|.
 
 </div>
 
+## SetHTML options and the configuration object. ## {#configobject}
+
+The family of {{Element/setHTML()}}-like methods always take an options
+dictionary. Right now, only one member of this dictionary is defined:
+
+<pre class=idl>
+dictionary SetHTMLOptions {
+  Sanitizer? sanitizer;
```

Comment:
This should not be nullable. And it should accept a dictionary directly as well. (Prolly needs to default to the empty dictionary therefore.)

Reply:
Done. (Also added a comment to the explainer.)